byzcoin/bcadmin/test.sh

```diff
@@ -236,4 +237,16 @@ testQR() {
   testOK ./"$APP" qr -admin
 }
 
+testUpdateDarcDesc() {
+  # We update the description of the latest darc, then we get the latest darc
+  # and check if the description changed.
+  runCoBG 1 2 3
+  runGrepSed "export BC=" "" runBA create --roster public.toml --interval .5s
+  eval $SED
+  [ -z "$BC" ] && exit 1
+
+  testOK runBA darc cdesc --desc "New description"
```

Comment:
Please make a new darc without evolve_unrestricted rights, and see that it can update it's own description.